docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -1,55 +1,41 @@
 ---
-id: authorize-the-create-endpoint
-title: Authorize the create endpoint
-description: Authorize the create endpoint
+id: 02-adding-a-basic-permission-check
+title: 2. Adding a basic permission check
+description: Explains how to add a basic permission check to a Backstage plugin
 ---
 
-The first step we need to do in order to authorize the create endpoint, is to create a new `permission` inside our backend plugin.
+If the outcome of a permission check doesn't need to change for different [resources](../concepts#resources-and-rules), you can use a _basic permission check_. For this kind of check, we simply need to define a [permission](../concepts#resources-and-rules), and call `authorize` with it.
```

Comment:
Change to intro copy in this PR aim to frame the specific work in each page in terms of the new title.